profiling-ffi/src/profiles.rs

```diff
@@ -325,12 +325,36 @@ pub unsafe extern "C" fn ddog_prof_Profile_drop(
 ) {
 }
 
+#[repr(C)]
+pub enum ProfileAddResult {
```

Comment:
A similar note to `ExporterRequestBuildResult` above applies here, with the final name being `ddog_prof_Profile_AddResult`